src/cli/serve/serve.js

```diff
@@ -20,6 +20,7 @@
 import _ from 'lodash';
 import { statSync } from 'fs';
 import { resolve } from 'path';
+import { CA_CERT_PATH } from '@kbn/dev-utils';
```

Comment:
are we okay shipping devcerts?  i know there's quite a bit in this file that makes it tough and extends beyond the scope of this PR

Reply:
Yeah, we should only be requiring this when `opts.dev` is true so that we don't have to ship `@kbn/dev-utils` in the distributable.